OsmIntegrator/ApiModels/Auth/ConfirmEmail.cs

```diff
@@ -6,13 +6,13 @@ public class ConfirmEmail
     {
         [Required]
         [EmailAddress]
-        public string? NewEmail { get; set; }
+        public string NewEmail { get; set; }
```

Comment:
Nie do konca jestem pewny co czy required  bez ? działa poprawnie. PRzy guidach nie działało

Reply:
Warning w vscode był raczej głupawy. kandydat do wyłączenia

Reply:
Chodziło o to, że w c# mamy typy proste takie jak int, bool, double, char, GUID oraz typy obiektowe takie jak object. Od object dziedziczą wszystkie obiekty, dlatego każdy ma domyślnie metodę ToString oraz GetHashCode , Equals. To jest bardzo podobne do Javy. String Jest specjalnym typem, bo w teorii powinien posiadać tylko tekst, a jednak również można przypisać mu null. W skrócie też jest obiektem.
Typy proste przechowywane są na stosie, który jest kolejką FIFO i do stosu jest generalnie szybszy dostęp, ale można pobrać tylko ostatnio dodaną wartość.
Obiekty przechowywane są na stercie, wtedy można pobrać dowolną wartość pod danym adresem, ale stera działa przez to wolniej.
Pytajnik ? przy typie prostym sprawia, że on staje się typem obiektowym i może przyjmować wartość null.
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/nullable-value-types
https://www.c-sharpcorner.com/article/stack-vs-heap-memory-c-sharp/